release/release_gold.sh

```diff
@@ -97,6 +98,252 @@ function main {
 	#lc_time_run upload_to_docker_hub
 
 	lc_time_run add_patcher_project_version
+
+	lc_time_run reference_new_releases
+}
+
+function reference_new_releases {
+	if [[ "${_PRODUCT_VERSION}" != *q* ]]
+	then
+		lc_log INFO "Skipping the update to the references in the liferay-jenkins-ee repository."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	fi
+
+	prepare_branch_to_commit_from_master "${_PROJECTS_DIR}/liferay-jenkins-ee/commands" "new_releases_branch"
+
+	if [ "${?}" -ne 0 ]
+	then
+		lc_log ERROR "Unable to prepare the next release references branch."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_BAD}"
+	fi
+
+	local previous_quarterly_release_branch_name="$(grep "extraapps.app.server.versions" "build.properties" | tail -1 | cut -d '[' -f 2 | cut -d ']' -f 1)"
+
+	if [[ "$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 3)" -eq 0 ]] && ! grep -q "${LIFERAY_RELEASE_GIT_REF}" "build.properties"
```

Comment:
@kiwm, can you create the `ifs` in a consistent way?

I mean, use single brackets ( if [ ... ] ) in both, for example. See other existing examples already available on the code.